apps/web/src/courses/test/challenges/animals.json

```diff
@@ -10,15 +10,6 @@
       "priority": 0,
       "group": "353b3be9ecc5"
     },
-    {
-      "type": "listeningExercise",
```

Comment:
@kantord is this to be expected? The now failing cypress tests, make me wonder. https://kantord.semaphoreci.com/jobs/aae1086d-13d9-48ae-9245-abbec15ca45b

Or do I just need to remove something from the cypress tests?

Reply:
this is weird, let me look into this

Reply:
I think it's kinda expected. my understanding is that we need to create the audio files that are missing. However, they can be literally empty text files as the export script won't be able to tell, it's only verifying that they exist afaik

Reply:
this is because now the exporting scripts should be able to handle missing audio by skipping listening exercises when the audio is missing. Hence it needs to be fetched first, but the test course won't have any way to fetch audio files.

The beautiful solution would be to add a new audio source called "Dummy audio" which will work the same way as the TTS audio source, but instead of fetching stuff form aws, it will just create dummy empty files.

Easy short term solution is to create these dummy files manually.

Reply:
@kantord thanks for the info. Where would the dummy files need to be added to? 
Also are you saying that if I add the dummy files and rerun `yarn exportAllCourses` will these `"type": "listeningExercise"` will not be deleted?

Reply:
>  Where would the dummy files need to be added to?

It would need to be added to this folder:
https://github.com/kantord/LibreLingo/tree/main/apps/web/static/voice

> Also are you saying that if I add the dummy files and rerun yarn exportAllCourses will these "type": "listeningExercise" will not be deleted?

yes

Reply:
Thank you, I will try to update this today.